gateway/handlers/alerthandler.go

```diff
@@ -100,14 +100,10 @@ func CalculateReplicas(status string, currentReplicas uint64, maxReplicas uint64
 	step := uint64((float64(maxReplicas) / 100) * float64(scalingFactor))
 
 	if status == "firing" && step > 0 {
-		if currentReplicas == 1 {
-			newReplicas = step
```

Comment:
The issue here is that `step` in rounded down (1.5 -> 1). I wonder if it's better to change the logic, or fix it by rounding up before casting.

Reply:
Great, just found 1 case that makes it fail.
When maxReplica*scaleFactor < 100. 

Just fixed, thanks @ivanayov

Reply:
If we fix by rounding up only, it still fail on new case I provide.
IMO, we still need to change the logic on line 103.

Reply:
ping @ivanayov

Reply:
I agree. The old logic is wrong for one more thin: we ignore `currentReplicas` when it's 1 and it's not proper to say `step` in that case